src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -688,6 +688,20 @@ public override Accessibility DeclaredAccessibility
             }
         }
 
+        public bool AreLocalsZeroed
+        {
+            get
+            {
+                var data = this.GetDecodedWellKnownAttributeData();
+                if (data == null)
+                {
+                    return ContainingType.AreLocalsZeroed;
+                }
+
+                return !data.HasSkipLocalsInitAttribute && ContainingType.AreLocalsZeroed;
```

Comment:
>!data.HasSkipLocalsInitAttribute && ContainingType.AreLocalsZeroed; [](start = 23, length = 67)

 Please add test when there is some other well-known attribute on the property and containing type has SkipLocalsInitAttribute

Reply:
The current state of the compiler does not support SkipLocalsInitAttribute applied directly on classes. That's why I left a prototype comment on `SourcePropertySymbolDelegatesToTypeWhenSkipLocalsInitAttributeIsNotFound` test. I fixed that in my current PR (#25673).